corehq/blobs/tests/test_migrate.py

```diff
@@ -0,0 +1,60 @@
+# coding=utf-8
+import json
+from os.path import join
+
+import corehq.blobs.migrate as mod
+from corehq.blobs.mixin import BlobMixin
+from couchexport.models import SavedBasicExport, ExportConfiguration
+from django.test import TestCase
+from testil import assert_raises, eq, tempdir
+
+
+class TestMigrations(TestCase):
+
+    def test_migrate_saved_exports(self):
+        # TODO generalize this to make it reusable for other slugs
+        slug = "saved_exports"
+
+        def _mk_config(name='some export name', index='dummy_index'):
+            return ExportConfiguration(index=index, name=name, format='xlsx')
+
+        # setup data
+        saved = SavedBasicExport(configuration=_mk_config())
+        saved.save()
+        payload = 'something small and simple'
+        name = saved.get_attachment_name()
+        super(BlobMixin, saved).put_attachment(payload, name)
+        saved.save()
+
+        # verify: attachment is in couch and migration not complete
+        eq(len(saved._attachments), 1)
+        eq(len(saved.external_blobs), 0)
+        with assert_raises(mod.BlobMigrationState.DoesNotExist):
+            mod.BlobMigrationState.objects.get(slug=slug)
+
+        with tempdir() as tmp:
+            filename = join(tmp, "file.txt")
+
+            # do migration
+            migrated = mod.MIGRATIONS[slug].migrate(filename)
+            assert migrated >= 1, migrated
```

Comment:
I didn't realize it was discouraged to use `assert` in tests. This seems clearer to me (and I can never remember all the `assertWhatever` methods).